common/keyboard.go

```diff
@@ -115,58 +116,68 @@ func (k *Keyboard) Type(text string, opts goja.Value) {
 	}
 }
 
-func (k *Keyboard) down(key string) error {
-	keyInput := keyboardlayout.KeyInput(key)
-	if _, ok := k.layout.ValidKeys[keyInput]; !ok {
-		return fmt.Errorf("%q is not a valid key for layout %q", key, k.layoutName)
-	}
+// Can pass in more than one key as long as they are separated by a `+`.
+func (k *Keyboard) down(keys string) error {
+	kk := strings.Split(keys, "+")
 
-	keyDef := k.keyDefinitionFromKey(keyInput)
-	k.modifiers &= ^k.modifierBitFromKeyName(keyDef.Key)
-	text := keyDef.Text
-	_, autoRepeat := k.pressedKeys[keyDef.KeyCode]
-	k.pressedKeys[keyDef.KeyCode] = true
+	for _, key := range kk {
+		keyInput := keyboardlayout.KeyInput(key)
+		if _, ok := k.layout.ValidKeys[keyInput]; !ok {
+			return fmt.Errorf("%q is not a valid key for layout %q", key, k.layoutName)
+		}
```

Comment:
I think you should do the `+` split and this key validation in `press()` instead, so that you don't have to repeat it, and `down()`/`up()` can continue handling single keys. It shouldn't be valid to call `down()`/`up()` from JS on a key combo anyway; that is what `press()` is for.

Also, if I'm reading this right, it will accept combos like `A+B+C`, which don't make sense. Maybe we should add validation that there should be only one non-modifier key pressed? Or should we just assume that `A+B+C` equals `ABC`? I guess we can fallback to whatever Playwright accepts, minus the `Shift+B`/`Shift+b` behavior.

Reply:
> And re: the `Shift+b` behavior, I don't think Playwright makes sense in that case. `Shift+b` or `Shift+B` should produce `B` in both cases. A lowercase `b` can be typed without `Shift`. It's fine if we diverge with them to improve things. :)

I'm mostly OK with what you propose @imiric, but here are my two cents.

Is diverging PW, in this case, worth it? If I were a tester, I'd at least expect similar behavior.  `Shift+KeyA+B+C` means holding down the shift key while pressing the other keys. For example: `Shift+KeyA+KeyB+B+b` is `ABBb`, which makes sense to me.

Puppeteers also work with `KeyA`, `KeyX`, etc. style.

So instead of `Shift+B`, it should be `Shift+KeyB`, not `Shift+B`. `KeyX` is also in tandem with JS key codes: https://javascript.info/keyboard-events. As I said, I'm also OK with what you proposed.

Reply:
Done